src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_AbstractInversionRunner.java

```diff
@@ -342,7 +383,10 @@ public FaultSystemSolution runInversion() throws IOException, DocumentException
 		tsa.setConstraintRanges(inversionInputGenerator.getConstraintRowRanges());
 		tsa.setRandom(new Random(1));
 		tsa.setRuptureSampler(null);
-		tsa.setPerturbationFunc(GenerationFunctionType.UNIFORM_NO_TEMP_DEPENDENCE);
+//		tsa.setPerturbationFunc(GenerationFunctionType.UNIFORM_NO_TEMP_DEPENDENCE);
+
+		tsa.setPerturbationFunc(perturbationFunction);
+//		tsa.setNonnegativeityConstraintAlgorithm(nonNegAlgorithm);		
```

Comment:
are you sure you don't want to use the new property here?

Reply:
bugger - well spotted thanks!

Reply:
yay for code reviews!